lib/public/AppFramework/Http/EmptyFeaturePolicy.php

```diff
@@ -35,6 +35,7 @@
  *
  * @see \OCP\AppFramework\Http\FeaturePolicy
  * @since 17.0.0
+ * @depreacted 21.0.0 Use \OCP\AppFramework\Http\EmptyPermissionPolicy
```

Comment:
```suggestion
 * @deprecated 21.0.0 Use \OCP\AppFramework\Http\EmptyPermissionPolicy
```

(+ more instances of `depreacted` -> `deprecated`)